Dockerfile

```diff
@@ -11,3 +11,4 @@ WORKDIR /home/builder/package
 ENV RSA_PRIVATE_KEY_NAME ssh.rsa
 ENV PACKAGER_PRIVKEY /home/builder/${RSA_PRIVATE_KEY_NAME}
 ENV REPODEST /packages
+VOLUME ["/home/builder/package", "$REPODEST"]
```

Comment:
Yes, you're right. The value is set at build time and doesn't get overridden when a container is created from the image. I'll remove the `$REPODEST` element.